imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -8,27 +8,30 @@
 logger = logging.getLogger(__name__)
 
 
-def mag_l1a(packet_filepath, output_filepath_norm, ouptput_filepath_burst):
+def mag_l1a(packet_filepath):
     """
     Process MAG L0 data into L1A CDF files at cdf_filepath.
 
     Parameters
     ----------
     packet_filepath :
         Packet files for processing
-    output_filepath_norm :
-        Full directory and filename for raw-norm CDF file
-    ouptput_filepath_burst :
-        Full directory and filename for raw-burst CDF file
     """
     mag_l0 = decom_mag.decom_packets(packet_filepath)
 
     mag_norm, mag_burst = decom_mag.export_to_xarray(mag_l0)
 
     if mag_norm is not None:
-        write_cdf(mag_norm, Path(output_filepath_norm))
-        logger.info(f"Created CDF file at {output_filepath_norm}")
+        file = write_cdf(mag_norm)
+        logger.info(f"Created CDF file at {file}")
 
     if mag_burst is not None:
-        write_cdf(mag_burst, Path(ouptput_filepath_burst))
-        logger.info(f"Created CDF file at {output_filepath_norm}")
+        file = write_cdf(mag_burst)
+        logger.info(f"Created CDF file at {file}")
+
+
+if __name__ == "__main__":
```

Comment:
Is this for local testing?

Reply:
yup, should not be in here. Thanks for catching!